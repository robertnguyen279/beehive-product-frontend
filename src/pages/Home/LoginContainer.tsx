import React from 'react';
import { HomeChild } from 'types/main';

const LoginContainer = ({ isLaptop }: HomeChild): React.ReactElement => {
  return isLaptop ? <div>laptop</div> : <div>mobile</div>;
};

export default LoginContainer;
