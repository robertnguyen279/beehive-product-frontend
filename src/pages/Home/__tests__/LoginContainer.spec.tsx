import React from 'react';
import { shallow } from 'enzyme';

import LoginContainer from '../LoginContainer';
// const clickFn = jest.fn();

describe('LoginContainer', () => {
  const component = shallow(<LoginContainer />);

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
