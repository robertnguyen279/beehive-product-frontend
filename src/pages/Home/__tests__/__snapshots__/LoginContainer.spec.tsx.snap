// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginContainer should render correctly 1`] = `
<div
  className="home__login-container flex md:absolute z-10"
>
  <div
    className="home__login-container__left md:w-1/2 h-full flex items-center justify-center sm:py-16 py-10"
  >
    <div
      className="w-4/5 text-white md:block flex flex-col items-center justify-center"
    >
      <div
        className="text-2xl font-bold font-title"
      >
        join-club
      </div>
      <div
        className="text-sm md:py-5 pt-2 pb-5"
      >
        join-club-description
      </div>
      <div
        className="flex items-center flex-col md:flex-row items-center pb-5 "
      >
        <div
          className="w-12 h-12 mr-5  flex items-center justify-center border border-white rounded"
        >
          <img
            alt="community"
            className="w-4/6"
            src="test-file-stub"
          />
        </div>
        <div
          className="text-center md:text-left"
        >
          <div
            className="font-bold text-lg font-title"
          >
            login-left-community
          </div>
          <div
            className="text-sm"
          >
            login-left-community-description
          </div>
        </div>
      </div>
      <div
        className="flex items-center flex-col md:flex-row items-center pb-5 "
      >
        <div
          className="w-12 h-12 mr-5 flex items-center justify-center border border-white rounded"
        >
          <img
            alt="online-shop"
            className="w-4/6"
            src="test-file-stub"
          />
        </div>
        <div
          className="text-center md:text-left"
        >
          <div
            className="font-bold text-lg font-title"
          >
            login-left-onlineshop
          </div>
          <div
            className="text-sm"
          >
            login-left-onlineshop-description
          </div>
        </div>
      </div>
      <div
        className="flex items-center flex-col md:flex-row items-center pb-5 "
      >
        <div
          className="w-12 h-12 mr-5 flex items-center justify-center border border-white rounded"
        >
          <img
            alt="job-search"
            className="w-4/6"
            src="test-file-stub"
          />
        </div>
        <div
          className="text-center md:text-left"
        >
          <div
            className="font-bold text-lg font-title"
          >
            login-left-jobsearch
          </div>
          <div
            className="text-sm"
          >
            login-left-jobsearch-description
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="md:w-1/2 h-full overflow-hidden bg-white flex flex-col items-center justify-center sm:py-16 py-10"
  >
    <div
      className="w-16 py-3"
    >
      <img
        alt="logo-icon"
        src="test-file-stub"
      />
    </div>
    <div
      className="text-2xl text-gray-600 font-bold font-title pb-3"
    >
      welcome
    </div>
    <div
      className="text-sm text-gray-600 pb-5"
    >
      join-with-people
    </div>
    <LoginRegisterForm />
  </div>
</div>
`;
